kong/templates/nginx_kong.lua

```diff
@@ -51,6 +51,11 @@ exit_worker_by_lua_block {
 }
 
 > if (role == "traditional" or role == "data_plane") and #proxy_listeners > 0 then
+log_format request_id_access_log_format '$remote_addr - $remote_user [$time_local] '
```

Comment:
We may name it as `kong_log_format`, and`log_format` directive can only be used for access_log, so that we dont add "_access_" string here.

From a perspective of troubleshooting, `$request_time`, `$upstream_status`, `$upstream_addr`, `$upstream_response_time`, this may not be appended to the log format currently, but I want to say that the access log is highly generic, so we should avoid assigning it a specific name such as `_request_id_`

Reply:
done thx